File: tests/fixtures/security/broadcast_receiver_exported_flag_missing/positive.java
package test;

import android.content.BroadcastReceiver;
import android.content.Context;
import android.content.Intent;
import android.content.IntentFilter;

class ReceiverSetup {
    void setup(Context context, BroadcastReceiver receiver) {
        context.registerReceiver(receiver, new IntentFilter(Intent.ACTION_SCREEN_ON));
        context.registerReceiver(receiver, new IntentFilter(Intent.ACTION_USER_PRESENT), 0);
    }
}

File: tests/fixtures/security/broadcast_receiver_exported_flag_missing/positive.kt
package test

import android.content.BroadcastReceiver
import android.content.Context
import android.content.Intent
import android.content.IntentFilter

class ReceiverSetup(private val context: Context, private val receiver: BroadcastReceiver) {
    fun setup() {
        context.registerReceiver(receiver, IntentFilter(Intent.ACTION_SCREEN_ON))
        context.registerReceiver(receiver, IntentFilter(Intent.ACTION_USER_PRESENT), 0)
    }
}
